DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/DelegateGroups/_SearchableDelegateGroupCard.cshtml

```diff
@@ -93,21 +93,21 @@
             Changes to field synchronised with group
           </dt>
           <dd class="nhsuk-summary-list__value">
-            @Model.SyncFieldChanges
+            @Model.SyncChangesToRegistrationInfoWithGroupMembership
           </dd>
           <dd class="nhsuk-summary-list__actions" />
         </div>
       </dl>
 
-      <a class="nhsuk-button left-button" role="button">
+      <a class="nhsuk-button nhsuk-button__card-button" role="button">

```

Comment:
Nice, think it's better not to start it with `nhsuk` though - people will assume it's an nhs style (or nhs might update their library and it might become an nhs style!). I appreciate the adherence to BEM, but I think in this case it should be a `delegate-card__button`, or maybe `searchable-element__button`?